# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../slim/fix_build.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- slim-1.3.1/switchuser.cpp.vanilla	2010-06-15 13:25:38.916930403 +0200
+++ slim-1.3.1/switchuser.cpp	2010-06-15 13:25:57.859843414 +0200
@@ -10,7 +10,7 @@
 */
 
 #include "switchuser.h"
-
+#include <stdio.h>
 using namespace std;
 
 SwitchUser::SwitchUser(struct passwd *pw, Cfg *c, const string& display,
